components/user-mgt/org.wso2.carbon.user.mgt.ui/src/main/java/org/wso2/carbon/user/mgt/ui/client/ClaimDataAdminClient.java

```diff
@@ -53,12 +55,44 @@ public ClaimDataAdminClient(String cookie, String backendServerURL, Configuratio
         option.setProperty(org.apache.axis2.transport.http.HTTPConstants.COOKIE_STRING, cookie);
     }
 
-    public LocalClaimDTO[] getLocalClaims() throws RemoteException, ClaimMetadataManagementServiceClaimMetadataException {
+    public LocalClaimDTO[] getLocalClaims()
+            throws RemoteException, ClaimMetadataManagementServiceClaimMetadataException {
 
         LocalClaimDTO[] localClaimDTOs = stub.getLocalClaims();
         if (localClaimDTOs == null) {
             localClaimDTOs = new LocalClaimDTO[0];
         }
         return localClaimDTOs;
     }
+
+    public String getRegex(LocalClaimDTO[] localClaimDTO, String claimURI) {
+
+        String pattern = "";
+        for (LocalClaimDTO localClaim : localClaimDTO) {
```

Comment:
Cant we use java 8 streaming here ?

Reply:
it is using non-final variables within the block. We can't use streams here since there are non-final variables.